docs/content/configuration/index.md

```diff
@@ -1176,6 +1176,8 @@ You can optionally configure caching to be enabled on the peons by setting cachi
 |`druid.realtime.cache.useCache`|true, false|Enable the cache on the realtime.|false|
 |`druid.realtime.cache.populateCache`|true, false|Populate the cache on the realtime.|false|
 |`druid.realtime.cache.unCacheable`|All druid query types|All query types to not cache.|`["groupBy", "select"]`|
+|`druid.realtime.cache.numBackgroundThreads`|If greater than 0, cache will be populated in the background thread pool of the configured size. By default cache is populated in the foreground, which can more efficiently handle reaching `maxEntrySize` than when done in the background. Note that there is no load shedding for background cache population, so it can also lead to out of memory scenarios depending on background threadpool utilization.|0|
+|`druid.realtime.cache.maxEntrySize`|Maximum cache entry size in bytes.|1_000_000|
```

Comment:
This description (or the more general description on the common page) should explain what happens if the serialized form of a query result is bigger than this size. I suppose it is "the result is not recorded in the cache; `XXX/put/oversized` metric is incremented" (I don't know what should go into XXX, this question should be researched.)

Reply:
This is what happens:

> the result is not recorded in the cache; cache `*/put/oversized` metrics are incremented.

The `*/put/oversized` syntax would match how they are described in metrics.md, so I think that'd be the right way to write it in user-facing docs.

The `*` could be `query/cache/delta` or `query/cache/total`, also mentioned in metrics.md.